### PR TITLE
[ilu/ttx] Add iluvatar group_rmsnorm triton impl.

### PR DESCRIPTION
**Pull request by madengfei**

### CODE REVIEW

File: mojo_opset/backends/ttx/kernels/ilu/group_rmsnorm.py [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+    )
+
+    return y_2d.reshape(T, H, N)
+
+
+def group_rmsnorm_impl(
+    input_groups,
+    weight=None,
+    eps=1e-6,
+    output_like_input_stride=True,
```
[the remark this answers is not in the document]
Comment (madengfei) [reply mojo_opset/backends/ttx/kernels/ilu/group_rmsnorm.py:133]: output_like_input_stride是为了保证input non-contiguous时，output的stride与之保持一致。但iluvatar platform尽可能使用contiguous tensor提升性能，将output_like_input_stride默认值设置为False。

File: mojo_opset/backends/ttx/kernels/ilu/group_rmsnorm.py [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+        wg = None if weight is None else weight[g]
+        if N != 0:
+            yg = _rmsnorm_fwd_single(
+                x=xg,
+                weight=wg,
+                eps=eps,
+            )
+        else:
+            yg = torch.empty_like(xg, memory_format=torch.contiguous_format)
+        output_groups.append(yg)
```
[the remark this answers is not in the document]
Comment (madengfei) [reply mojo_opset/backends/ttx/kernels/ilu/group_rmsnorm.py:168]: 先提供Naive实现，后续针对具体case优化。

File: mojo_opset/backends/ttx/kernels/ilu/group_rmsnorm.py [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+
+    x_contig = x.contiguous()
+    x_2d = x_contig.reshape(-1, N)
+    y_2d = torch.empty_like(x_2d)
+    n_rows = x_2d.shape[0]
+
+    if N > COL_BLOCKING_THRESHOLD:
+        BLOCK_SIZE_N = COL_BLOCKING_THRESHOLD
+    else:
+        BLOCK_SIZE_N = align(x_2d, N, VEC_ALIGN_BYTES)
```
[the remark this answers is not in the document]
Comment (madengfei) [reply mojo_opset/backends/ttx/kernels/ilu/group_rmsnorm.py]: 已修改

File: mojo_opset/backends/ttx/kernels/ilu/group_rmsnorm.py [hunk 4]
```diff
@@ -0,0 +1,151 @@
+import torch
+
+import triton
+import triton.language as tl
+
+from .utils import COL_BLOCKING_THRESHOLD
+from .utils import VEC_ALIGN_BYTES
+from .utils import ilu_grid_dim_from_row_tasks
+from .utils import rms_norm_fwd_heuristics
```
[the remark this answers is not in the document]
Comment (madengfei) [reply mojo_opset/backends/ttx/kernels/ilu/group_rmsnorm.py:9]: 已修改

File: mojo_opset/backends/ttx/operators/normalization.py [hunk 5]
```diff
@@ -10,13 +10,15 @@
...
 from mojo_opset.core import MojoRMSNorm
 from mojo_opset.core import MojoGroupRMSNorm
+from mojo_opset.utils.platform import get_platform
 
 class TTXGroupRMSNorm(MojoGroupRMSNorm):
-    supported_platforms_list = ["mlu"]
+    supported_platforms_list = ["mlu", "ilu"]
 
     def forward(self, input_groups) -> list[torch.tensor]:
+        output_like_input_stride = get_platform() != "ilu"
```
Comment (wwens7) [inline mojo_opset/backends/ttx/operators/normalization.py]: 这里是啥背景

Comment (songliucsu) [reply mojo_opset/backends/ttx/operators/normalization.py]: 错误修改，fix

**Review by wwens7 — APPROVED** [review]
